src/plotloader/ScatterPlot.js

```diff
@@ -117,6 +120,28 @@ const highchartsBaseConfig = {
       turboThreshold: 0,
       animation: false
     }
+  },
+
+  navigation: {
+    buttonOptions: {
+      theme: {
+        style: {
+          fontSize: `15px`
+        }
+      }
+    },
+    menuItemStyle: {
+      fontSize: '15px'
+    }
+  },
+
+  exporting: {
+    buttons: {
+      contextButton: {
+        text: '<i class="icon icon-functional" data-icon="="></i> Download &nbsp;',
```

Comment:
The non-breaking space should be placed between the `<i>` element and the Download word, not the other way around.